<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HospDash</class>
 <widget class="QMainWindow" name="HospDash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>1181</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0.153409 rgba(0, 145, 174, 255), stop:1 rgba(184, 50, 50, 255));
border-radius: 30px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushVirtualdoctor">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>40</y>
       <width>341</width>
       <height>221</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Virual Doctor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushAppointment">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>40</y>
       <width>351</width>
       <height>221</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Book an Appointment</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushHospitals">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>341</width>
       <height>221</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Availabe Hospitals</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushDashboard">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>430</y>
       <width>341</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Back to DashBoard</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
